crates/wdk-build/src/cargo_make.rs

```diff
@@ -1234,28 +1234,148 @@ pub fn driver_sample_infverif_condition_script() -> anyhow::Result<()> {
 
 #[cfg(test)]
 mod tests {
-    use crate::ConfigError;
-
-    const WDK_TEST_OLD_INF_VERSION: &str = "10.0.22061.0";
-    const WDK_TEST_NEW_INF_VERSION: &str = "10.0.26100.0";
-
-    #[test]
-    fn check_env_passing() -> Result<(), ConfigError> {
-        crate::cargo_make::setup_infverif_for_samples(WDK_TEST_OLD_INF_VERSION)?;
-        let env_string = std::env::var_os(crate::cargo_make::WDK_INF_ADDITIONAL_FLAGS_ENV_VAR)
-            .map_or_else(
-                || panic!("Couldn't get OS string"),
-                |os_env_string| os_env_string.to_string_lossy().into_owned(),
-            );
-        assert_eq!(env_string.split(' ').next_back(), Some("/msft"));
+    mod setup_infverif_for_samples {
```

Comment:
This test is the same as before. I've just put it into a module named after the function it's testing (a convention we follow elsewhere in the repo)